api/ccapi/include/layer.h

```diff
@@ -50,6 +50,7 @@ enum LayerType {
   LAYER_EMBEDDING,                            /** Embedding Layer type */
   LAYER_RNN,                                  /** RNN Layer type */
   LAYER_LSTM,                                 /** LSTM Layer type */
+  LAYER_TIME_DIST,                            /** Time Distributed Layer type */
```

Comment:
if this layer can be constructed and add to the model manually by the user, just keep in mind that this will need special handling in the graph, which can be done in a later PR.

Reply:
This will come in next pr. #1074